api/bases/test.openstack.org_horizons.yaml

```diff
@@ -0,0 +1,184 @@
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.11.1
+  creationTimestamp: null
+  name: horizons.test.openstack.org
+spec:
+  group: test.openstack.org
+  names:
+    kind: Horizon
```

Comment:
I'm writing it here so that we do not forget about this. 

We should use a different name for the CR that can be used for horizon testing. Maybe something like `HorizonTest`? But we probably shouldn't use the same name as for the Horizon service.